prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -213,31 +213,43 @@ void rxr_pkt_init_req_hdr(struct rxr_ep *ep,
 	assert(opt_hdr - pkt_entry->pkt == rxr_pkt_req_hdr_size(pkt_entry));
 }
 
+/**
+ * @brief Get the value of rma_iov_count in req pkt header.
+ *
+ * @param[in] pkt_entry the pkt entry
+ * @return The rma_iov_count in the pkt header, if it is part of the header.
+ * Otherwise return 0.
+ */
+uint32_t rxr_pkt_req_hdr_rma_iov_count(struct rxr_pkt_entry *pkt_entry)
```

Comment:
Do you think it would be better to use this method?

https://github.com/ofiwg/libfabric/blob/a2120d6fc633abb76f158ca4fce79a0a80a62a70/prov/efa/src/rxr/rxr_pkt_hdr.h#L48

Reply:
Could you elaborate? How can I use this method?

Reply:
something like 

```
static const init offset_of_rma_iov_in_hdr[] = {
     [EAGER_RTW] = offset_of(rma_iov_count, struct rxr_eager_rtw_hdr)
    ....
};

int *rma_iov_pointer = (int *)(pkt_entry->pkt + offset_of_rma_iov_count_in_hdr[pkt_type];
return *rma_iov_pointer;
```

Reply:
Cool, will do.

Reply:
Think about it again, I do not think this method simplifies the function. So I would keep the current implementation.